docs/blog/_posts/2019-12-20-21th-dotty-milestone-release.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.21.0-RC1 – ❄️ Feature Freeze ❄️
+author: Aggelos Biboudis
+authorImg: /images/aggelos.png
+date: 2019-12-21
+---
+
+Greetings and we wish you Merry Christmas 🎄!
+
+We are excited to announce 0.21.0-RC1 of Scala 3. In this version we add support
+for non-nullable reference types and a flow-sensitive analysis that work
+complementary. We present new syntax for given extensions, matches and pattern
+matching over quotes. We are also happy to announce that SemanticDB extraction
+is now supported!
+
+# ❄️ Feature Freeze ❄️
+This release is a HUGE milestone for us, for Dotty, for Scala 3, for our community. Since that
+[initial commit](https://github.com/lampepfl/dotty/commit/90962407e72d88f8f3249ade0f6bd60ff15af5ce)
+on the 6th December of 2012 when the only feature was the basic structure of a
+compiler based on the DOT calculus, we have come a long way.
+
+7 years and 20k commits later we are happy to announce that we are entering the
+*feature freeze* phase of Scala 3 ❄️❄️☃️. This means that with this release we
+stop adding new features ([Overview of Features](https://dotty.epfl.ch/docs/reference/overview.html)) and we focus on:
+
+- bug fixing and general quality assurance towards the final release 🐛
+- performance engineering 🏎️
+- documentation improvements 📕
+- education 👨‍🏫
+
```

Comment:
Add: 

Being feature complete does not mean that every detail of Scala-3 is cast in stone yet. Some details can still change, or be enabled conditionally, or even be dropped entirely. That will depend on the additional experience we gain over the next year, in particular the feedback we receive from the community and the SIP committee. But the envelope of what will (most likely) be in Scala 3 is now defined and implemented.

@odersky	Reply…